test/intratileopt5.c

```diff
@@ -1,31 +1,28 @@
-// CHECK: T(S1): (i, j, 0, 1, 0)
-// CHECK: T(S2): (i, j, 1, 0, k)
-// CHECK: T(S3): (i, j, 0, 0, 0)
-// CHECK: T(S4): (i, j+k, 1, 1, k)
-// CHECK-TILE: After tiling:
-// CHECK-TILE: T(S1): (i/32, j/32, 0, 1, 0, i, j, 0, 1, 0)
-// CHECK-TILE: T(S2): (i/32, j/32, 1, 0, k/32, i, j, 1, 0, k)
-// CHECK-TILE: T(S3): (i/32, j/32, 0, 0, 0, i, j, 0, 0, 0)
-// CHECK-TILE: T(S4): (i/32, (j+k)/32, 1, 1, k/32, i, j+k, 1, 1, k)
+// CHECK: T(S1): (0, i, j, 1, 0)
+// CHECK: T(S2): (1, i, j, 0, k)
+// CHECK: T(S3): (0, i, j, 0, 0)
+// CHECK: T(S4): (1, i, k, 1, j)
```

Comment:
This looks great! - that the unnecessary skewing is gone and the right loop is innermost.